networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/mock/NoOpEth2Network.java

```diff
@@ -27,4 +28,9 @@ public void unsubscribeFromAttestationSubnetId(final int subnetId) {}
 
   @Override
   public void setLongTermAttestationSubnetSubscriptions(final Iterable<Integer> subnetIndices) {}
+
+  @Override
+  public MetadataMessage getMetadata() {
+    return null;
```

Comment:
We should probably make this return `Optional` or this is going to create a lot of surprises when p2p is disabled.

Reply:
Alternatively this can probably just return sequence 1 and all zeros for the subscribed subnets which is probably better.

Reply:
ok, there's a MetadataMessage.createDefault() which creates sequence 0 and all 0's for subscribed, so i've called that here. hopefully sequence 0 is fine...

Reply:
Yep, using that is a better idea. :)